modules/labs-react/common/spec/useThemeRTL.spec.ts

```diff
@@ -138,8 +138,7 @@ describe('styles', () => {
 
         const {result} = renderHook(() => useThemeRTL());
         const cssProp = result.current.themeRTL(style);
-
-        expect((cssProp['&.hover'] as CSSObject).paddingLeft).toEqual(expectedPadding);
+        expect(cssProp['&.hover'].paddingLeft).toEqual(expectedPadding);
```

Comment:
Having troubles with this file, does anyone have time to look into this test failure or help me figure it out?